src/modules/pages/AuthorizationCallback/AuthorizationCallback.js

```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux';
+
+import { getJWTtoken } from '../../../store/actions';
+
+const AuthorizationCallback = props => {
+    props.getJWTtoken(document.location.hash);
+
+    return null;
+};
+
+export default connect(
+    null, // mapStateToProps
```

Comment:
?

Reply:
In this case, I do not need anything from _state_, that's why `null` was passed